src/app/subscriptions/views/SubscriptionSummaryConfigView.vue

```diff
@@ -6,36 +6,28 @@
       codeFilter: false,
       codeRegExp: false,
     }"
-    v-slot="{ route, t }"
+    v-slot="{ route }"
   >
     <AppView>
-      <template #title>
-        <h2>
-          {{ t('subscriptions.routes.item.headers.config') }}
-        </h2>
-      </template>
-
-      <KCard>
-        <XCodeBlock
-          language="yaml"
-          :code="YAML.stringify(props.data)"
-          is-searchable
-          :query="route.params.codeSearch"
-          :is-filter-mode="route.params.codeFilter"
-          :is-reg-exp-mode="route.params.codeRegExp"
-          @query-change="route.update({ codeSearch: $event })"
-          @filter-mode-change="route.update({ codeFilter: $event })"
-          @reg-exp-mode-change="route.update({ codeRegExp: $event })"
-        />
-      </KCard>
+      <XCodeBlock
+        language="yaml"
+        :code="YAML.stringify(props.data.$raw)"
+        is-searchable
+        :query="route.params.codeSearch"
+        :is-filter-mode="route.params.codeFilter"
+        :is-reg-exp-mode="route.params.codeRegExp"
+        @query-change="route.update({ codeSearch: $event })"
+        @filter-mode-change="route.update({ codeFilter: $event })"
+        @reg-exp-mode-change="route.update({ codeRegExp: $event })"
+      />
```

Comment:
I noticed the SubscriptionConfig tab content had a header and a KCard which is not how we render this in other areas of the GUI so I removed those things